packages/aserto/test/test_directory.py

```diff
@@ -134,9 +133,7 @@ def test_get_objects_many(directory):
             ObjectIdentifier(key=directory.obj_2.key, type=directory.obj_2.type),
         ]
     )
-    assert directory.obj_1 in objs
-    assert directory.obj_2 in objs
-    assert directory.obj_3 not in objs
+    assert len(objs) == 2
```

Comment:
Same here. Only the third assertion (`directory.obj_3 not in objs`) is superseded by the length check, not the first two. Together with the length check, the test verifies that `obj_1` and `obj_2` are the only elements in `objs`.
You could also write it in a single assert using sets:
```python
assert set(objs) == {directory.obj_1, directory.obj_2}
```

You can think of sets as dictionaries that don't have values, only key. This means that there are no repetitions and order doesn't matter. In other words `{a, b, a} == {b, a}`.